src/vm/analysis/tests/costs.rs

```diff
@@ -198,33 +194,35 @@ pub fn test_tracked_costs(prog: &str, use_mainnet: bool, epoch: StacksEpochId) -
     }
 }
 
-fn test_all(use_mainnet: bool) {
-    let baseline = test_tracked_costs("1", use_mainnet, StacksEpochId::Epoch20);
+fn epoch_21_test_all(use_mainnet: bool) {
+    let baseline = test_tracked_costs("1", use_mainnet, StacksEpochId::Epoch21);
 
     for f in NativeFunctions::ALL.iter() {
         let test = get_simple_test(f);
-        let cost = test_tracked_costs(test, use_mainnet, StacksEpochId::Epoch20);
+        let cost = test_tracked_costs(test, use_mainnet, StacksEpochId::Epoch21);
         assert!(cost.exceeds(&baseline));
     }
 }
 
 #[test]
-fn test_all_mainnet() {
-    test_all(true)
+fn epoch_21_test_all_mainnet() {
+    epoch_21_test_all(true)
```

Comment:
Can you make this merge additive, so that there's `epoch_21_test_all`, `epoch_20_test_all` and `epoch_205_test_all` in both mainnet and testnet versions?